substrate/client/rpc-spec-v2/Cargo.toml

```diff
@@ -36,6 +36,7 @@ tokio = { version = "1.22.0", features = ["sync"] }
 array-bytes = "6.1"
 log = "0.4.17"
 futures-util = { version = "0.3.19", default-features = false }
+impl-serde = { version = "0.4.0", default-features = false }
```

Comment:
Would it still be necessary to have this extra dependency if we use a `serde(from = ...` and `serde(into = ` for the `RuntimeVersion`? 🤔

Reply:
I think we would still need it, because we would still need to define how `ApiId` (which is a `[u8; 8]`) is serialized (as a string). The serialization of  `RuntimeVersion` also relies on a custom serialization using `impl-serde`, so we would ahve to do something similar for our wrapper.